README.md

```diff
@@ -64,10 +66,26 @@ __Setup Instructions__
 
 For more information about setting up a Service Account look at the [google-spreadsheet npm package](https://github.com/theoephraim/node-google-spreadsheet)
 
-Multiple values are possible for the column names where the values can be found. This way you can use the translation file for multiple platforms. The array should contain the column names sorted by importance. If there is no value found in the first column it will fall back to the next and so on ... 
+Multiple values are possible for the column names where the values can be found. This way you can use the translation file for multiple platforms. The array should contain the column names sorted by importance. If there is no value found in the first column it will fall back to the next and so on ...
 
 In the column names for the values you can specify the language by adding ```$$LANG$$``` this will be replaced by the actual language. For example: ```ios fallback $$LANG$$``` will become ```ios fallback en```.
 
+### Syncing from an endpoint that returns strings files
+
+```json
+{
+  "sync": {
+    "provider": "url",
+    "config": {
+      "url": "https://some.remote.end/point/{{LANG}}.strings",
```

Comment:
Perhaps $$LANG$$ makes more sense, equivalence with how column-names for spreadsheets work?
Other than that, what's the formatting issue here?